Units/parser-javascript.r/js-commas-and-missing-semicolons.d/expected.tags

```diff
@@ -0,0 +1,11 @@
+A	input.js	/^var A = {}, b = 2;$/;"	v
+b	input.js	/^var A = {}, b = 2;$/;"	v
+c	input.js	/^var c = 3, d = 4;$/;"	v
+d	input.js	/^var c = 3, d = 4;$/;"	v
+mem1	input.js	/^A.mem1 = function() {return 42;}, A.mem2 = function() {return 43;}$/;"	f	function:A
```

Comment:
The problem is that in JavaScript there's not really anything but objects, a class, a function, no real differences.  Actually you can see a fun thing in Firefox's JS console:
```
>> class foo {}
undefined
>> print(foo)
"function foo() {
    
}"
```

Anyway, my point is that the effective high-level kind is very dynamic.  In this example, A should be a class merely because it gets methods added, but it really becomes a class.  So ideally A should be reported as a class, but when it's first seen and the tag emitted it looks like a naked object, hence the "variable" kind.